docs/source/examples/de_sscan.ipynb

```diff
@@ -0,0 +1,1542 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The synApps `sscan`\n",
+    "\n",
+    "The synApps `sscan` record is used to measure scans of detector(s) *v*. positioner(s).  \n",
+    "The `sscan` record has three different scan modes:\n",
+    "\n",
+    "`sscan` scan mode | bluesky plan | description\n",
+    "--- | --- | ---\n",
+    "linear | `scan()` | step scan with constant-size steps\n",
+    "table | `list_scan()` | step scan with list of positions\n",
+    "fly | `fly()` | continuous motion scan\n",
+    "\n",
+    "**Goals**: Demonstrate use of the `sscan` record with Bluesky.\n",
+    "\n",
+    "1. Press SCAN button of a preconfigured scan.\n",
+    "\n",
+    "    1. Use a polling loop to wait for scan to end.\n",
+    "    2. Use a Status object to wait for scan to end.\n",
+    "\n",
+    "2. Same, but connect with most `sscan` record fields and get data after scan.\n",
```

Comment:
This sentence is unclear, what do you mean by "most"?
I would suggest something like:

"Same as 1B, but uses an instance of the `apstools.synApps.SscanRecord` class instead of the generic `ophyd.EpicsSignal` class."

Reply:
I'll add links to documentation of each.